packages/cli/src/commands/secrets.js

```diff
@@ -199,6 +211,11 @@ async function run({ output, token, contextName, currentTeam, client }) {
   }
 
   if (subcommand === 'rm' || subcommand === 'remove') {
+    output.note(
+      `The ${getCommandName(
+        'env rm'
```

Comment:
This makes it sound like the commands are interchangeable, but I don't think it's possible to delete secrets with env. So this might be a bit confusing

Reply:
Maybe it'd make sense to mention that `vc secrets` is a deprecated flow in general and that `vc env` is recommended.

Reply:
To attach a link would be a good idea @AndyBitz